public/controllers/agent/register-agent/steps/wz-manager-address.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { memo, useCallback, useEffect, useState } from 'react';
 import { EuiText, EuiFieldText } from '@elastic/eui';
+import _ from '@lodash';
+// import { usePrevious } from '../../../../../public/components/common/hooks/use-previous';
```

Comment:
**nitpick**: remove the comment if it is not required.

Reply:
Deleted in commit [c0fed1d](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/c0fed1da1dde3abb36fec3b527689b70d48f293f)